rubin_sim/utils/spatial_bounds.py

```diff
@@ -31,7 +30,7 @@ def __init__(cls, name, bases, dct):
         return super(SpatialBoundsMetaClass, cls).__init__(name, bases, dct)
 
 
-class SpatialBounds(with_metaclass(SpatialBoundsMetaClass, object)):
+class SpatialBounds(metaclass=SpatialBoundsMetaClass:
```

Comment:
Can this really run at all? It looks like the parenthesis are unbounded.

Reply:
Thank you .. that's what you get for editing quickly in emacs in the terminal.